src/FixedPointDecimals/FixedPointDecimals.jl

```diff
@@ -220,17 +220,10 @@ realmin{T <: FD}(::Type{T}) = eps(T)
 realmax{T <: FD}(::Type{T}) = typemax(T)
 
 # printing
-function show{T}(io::IO, x::FD{T, 0})
-    iscompact = get(io, :compact, false)
-    if !iscompact
-        print(io, "FixedDecimal{$T,0}(")
-    end
+function print{T}(io::IO, x::FD{T, 0})
     print(io, x.i)
-    if !iscompact
-        print(io, ')')
-    end
 end
-function show{T, f}(io::IO, x::FD{T, f})
+function print{T, f}(io::IO, x::FD{T, f})
     iscompact = get(io, :compact, false)
```

Comment:
With other types, `print` generally doesn't print trailing zeros, while the default `compact` behavior here does. It's debatable which behavior makes more sense. I do like the idea of printing trailing zeros (as you've done here) as it does give indication that those zeros are "stored", as opposed to something rounded to.